torch/utils/data/datapipes/iter/__init__.py

```diff
@@ -1,7 +1,9 @@
 from torch.utils.data.datapipes.iter.listdirfiles import ListDirFilesIterDataPipe as ListDirFiles
 from torch.utils.data.datapipes.iter.loadfilesfromdisk import LoadFilesFromDiskIterDataPipe as LoadFilesFromDisk
-from torch.utils.data.datapipes.iter.readfilesfromtar import ReadFilesFromTarIterDataPipe as ReadFilesFromTar
-from torch.utils.data.datapipes.iter.readfilesfromzip import ReadFilesFromZipIterDataPipe as ReadFilesFromZip
+from torch.utils.data.datapipes.iter.openfilesfromarchive import \
+    (OpenFilesFromArchiveIterDataPipe as OpenFilesFromArchive,
+     OpenFilesFromTarIterDataPipe as OpenFilesFromTar,
+     OpenFilesFromZipIterDataPipe as OpenFilesFromZip)
```

Comment:
Just curious: why not go with the parentheses on the first and last line?

```suggestion
from torch.utils.data.datapipes.iter.openfilesfromarchive import (
    OpenFilesFromArchiveIterDataPipe as OpenFilesFromArchive,
    OpenFilesFromTarIterDataPipe as OpenFilesFromTar,
    OpenFilesFromZipIterDataPipe as OpenFilesFromZip,
)
```


This makes it a lot easier to modify a single item.